data/human/human missions.txt

```diff
@@ -7497,3 +7497,211 @@ mission "Adelita 5"
+			choice
+				`	"Sure, I'll invest. What do I need to do?"`
+				`	"Nope, I'll stick with the 90k."`
+					goto payup
+			label invest
+			`	"Excellent. I'll need to get approval for the bond sale and set up the paperwork, so the deal won't be ready today, but feel free to come back to my kiosk here at the spaceport anytime afterwards to execute the purchase. Remember, you'll need to have at least one million credits for the bond purchase and to take care of a few other administrative tasks."`
+				decline
+			label payup
+			`	"Erhm, yes, of course. Here you are, my savings of ninety-one thousand, nine hundred and fifty one credits. I remain in your debt, and if you still want to invest in us, I will be here."`
+			action
+				payment 91951
+				set "syndicate business: mean to Howard"
+
+event "start of independent violence"
+	government "Independent (Killable)"
+		"attitude toward"
+			"Independent" -.2
+	government "Independent"
+		"attitude toward"
+			"Independent (Killable)" -.2
+
+event "end of independent violence"
+	government "Independent (Killable)"
+		"attitude toward"
+			"Independent" 0
+	government "Independent"
+		"attitude toward"
+			"Independent (Killable)" 0
+
+# Feel free to change this escape pod to a different ship once escape pod mechanics are implemented.
+ship "Escape Pod"
+	sprite "ship/escape pod"
+	attributes
+		category "Utility"
+		"cost" 45000
+		"shields" 300
+		"hull" 600
+		"required crew" 1
+		"bunks" 10
+		"mass" 65
+		"drag" 1.7
+		"heat dissipation" .7
+		"fuel capacity" 100
+		"cargo space" 20
+		"outfit space" 39
+		"weapon capacity" 0
+		"engine capacity" 20
+		weapon
+			"blast radius" 9
+			"shield damage" 90
+			"hull damage" 45
+			"hit force" 135
+	outfits
+		"KP-6 Photovoltaic Panel" 2
+		"LP036a Battery Pack"
+		"Outfits Expansion"
+		
+		"X1050 Ion Engines"
+
+		"Hyperdrive"
+		
+	engine 0 26
+		zoom 1.4
+	leak "leak" 40 50
+	explode "tiny explosion" 5
+	explode "small explosion" 1
+	description "This escape pod is a custom-built Shuttle, designed to be attached to a larger ship and to serve as a lifeboat if the need arises. It is intentionally designed to maximize the number of people it can carry to safety, but has very little space for outfits. While escape pods usually do not come with a hyperdrive installed, this one has been modified in order to fit one in."
+
+
+
+mission "Syndicate Business 2"
+	name "Purchase bond on <planet>"
+	description "Bring Howard to <destination> in order to finish the paperwork you'll need to finalize your bond agreement with Megaparsec Incorporated."
+	passengers 1
+	source "Sunracer"
+	destination "Hephaestus"
+	to offer
+		has "Syndicate Business 1: done"
+		credits > 1000000
```

Comment:
This ought to be in a "to accept" condition and the mission should have a "blocked" message. The previous message could also mention the need to transport Howard somewhere, which should be repeated in the "blocked" message, else a person without the passenger space will just not get this mission for no apparent reason.

Reply:
This is a good callout -- though a bit new to me so I'll take a little time to playtest to make sure it works as expected.